packages/errpy/errpy.0.0.10/opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Errpy: An Error Recovering Python Parser implemented in Rust"
+description:
+  "This is a new project to provide an error-recovering Python parser, implemented in Rust based on tree-sitter. Our short-term goal is to use it in the pyre type checker, in order to provide better IDE features."
+maintainer: ["pyre@meta.com"]
+authors: ["pyre@meta.com"]
+license: "MIT"
+homepage: "https://github.com/facebook/errpy"
+bug-reports: "https://github.com/facebook/errpy/issues"
+depends: [
+  "ocaml"
```

Comment:
```suggestion
  "ocaml" {< "5.0"}
```
Due to
```
#=== ERROR while compiling errpy.0.0.10 =======================================#
# context              2.2.0~alpha3~dev | linux/x86_64 | ocaml-base-compiler.5.1.0 | pinned(https://github.com/facebook/errpy/archive/refs/tags/0.0.10.tar.gz)
# path                 ~/.opam/5.1/.opam-switch/build/errpy.0.0.10
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p errpy -j 255 @install
# exit-code            1
# env-file             ~/.opam/log/errpy-7-2037ce.env
# output-file          ~/.opam/log/errpy-7-2037ce.out
### output ###
# File "vendor/ocaml/interop/ocamlrep_ocamlpool/dune", line 8, characters 9-18:
# 8 |   (names ocamlpool)))
#              ^^^^^^^^^
# (cd _build/default/vendor/ocaml/interop/ocamlrep_ocamlpool && /usr/bin/gcc -O2 -fno-strict-aliasing -fwrapv -pthread -fPIC -D_FILE_OFFSET_BITS=64 -g -I /home/opam/.opam/5.1/lib/ocaml -o ocamlpool.o -c ocamlpool.c)
# ocamlpool.c: In function 'ocamlpool_reserve_block':
# ocamlpool.c:383:14: error: too few arguments to function 'caml_shared_try_alloc'
#   383 |   value* p = caml_shared_try_alloc(d->shared_heap, wosize, tag, 0);
#       |              ^~~~~~~~~~~~~~~~~~~~~
# In file included from ocamlpool.c:296:
# /home/opam/.opam/5.1/lib/ocaml/caml/shared_heap.h:33:8: note: declared here
#    33 | value* caml_shared_try_alloc(struct caml_heap_state*,
#       |        ^~~~~~~~~~~~~~~~~~~~~
```

Reply:
Ah ok thanks, I will have re resubmit the PR to resolve that